### ISSUE TITLE
Serialization dumps with static references instead of paths in 1.39.0

### ISSUE THREAD

**Issue by flamber** [metabase/metabase#15812]
**Describe the bug**
Serialization does not handle the new field dimension(instead of `field-id`), so it fails to replace references during dump

**To Reproduce**
1. Create a Simple question on the source like Orders: Sum of Total group by CreatedAt:Month
2. Serialization dump - no errors reported
3. The generated yaml looks like this with static ID references
```
...
dataset_query:
  type: query
  query:
    source-query:
      source-table: /databases/Sample Dataset/schemas/PUBLIC/tables/ORDERS
      aggregation:
      - - sum
        - - field
          - 15
          - null
      breakout:
      - - field
        - 12
        - temporal-unit: month
...
```

**Expected behavior**
This is 1.38.4:
```
...
dataset_query:
  type: query
  query:
    source-query:
      source-table: /databases/Sample Dataset/schemas/PUBLIC/tables/ORDERS
      aggregation:
      - - sum
        - - field-id
          - /databases/Sample Dataset/schemas/PUBLIC/tables/ORDERS/fields/TOTAL
      breakout:
      - - datetime-field
        - - field-id
          - /databases/Sample Dataset/schemas/PUBLIC/tables/ORDERS/fields/CREATED_AT
        - month
...
```

**Information about your Metabase Installation:**
1.39.0.1

**Comment by jeff303** [comment]
This should be fixed by commit 1 on #15858.

**Comment by flamber** [comment]
Fixed by #15858 - will be part of 1.39.3
